applications/rag/example_notebooks/rag-data-ingest-with-kubernetes-docs.ipynb

```diff
@@ -0,0 +1,468 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
```

Comment:
What's the id used for

Reply:
@gongmax it's some metadata that Jupyter gives to the blocks.